packages/react/src/themes/teams-high-contrast/components/Button/buttonVariables.ts

```diff
@@ -10,25 +10,16 @@ export default (siteVars: any): Partial<ButtonVariables> => {
     backgroundColorFocus: siteVars.accessibleYellow,
     backgroundColorDisabled: siteVars.accessibleGreen,
     borderColor: siteVars.white,
-    borderColorActive: 'transparent',
     borderColorHover: 'transparent',
-    borderColorFocus: siteVars.black,
-    borderColorFocusIndicator: siteVars.white,
 
     primaryColor: siteVars.white,
-    primaryColorActive: siteVars.black,
     primaryColorHover: siteVars.black,
     primaryColorFocus: siteVars.black,
     primaryBackgroundColor: siteVars.black,
```

Comment:
I'm not sure if this is the right value to change, but I believe the primary rest background color for buttons in Teams hc should be white
![image](https://user-images.githubusercontent.com/828692/57274710-e862e280-7050-11e9-82b2-2ef42f044203.png)

Reply:
resolving this and tracking separately in #1300